content/docs/authorization/cli.md

```diff
@@ -1095,3 +1096,42 @@ karavictl tenant delete [flags]
 $ karavictl tenant delete --name Alice
 ```
 On success, there will be no output. You may run `karavictl tenant get --name <tenant-name>` to confirm the deletion occurred.
+
+
+
+---
+
+
+
+### karavictl tenant update
+
+Updates a tenant's approvesdc boolean flag value within CSM only for Powerflex Pre-approved Guid feature
```

Comment:
Make this a generic command description for tenant update, this could also support other features in the future.

Reply:
Made the changes.
This is the doc PR for Powerflex driver settings for this feature
https://github.com/dell/csm-docs/pull/455